pkg/apiclient/apiclient.go

```diff
@@ -234,6 +234,12 @@ func NewClient(opts *ClientOptions) (Client, error) {
 	}
 	if opts.AuthToken != "" {
 		c.AuthToken = opts.AuthToken
+		parser := jwt.NewParser(jwt.WithoutClaimsValidation())
+		claims := jwt.RegisteredClaims{}
+		_, _, err = parser.ParseUnverified(c.AuthToken, &claims)
+		if err != nil {
+			return nil, status.Error(codes.InvalidArgument, err.Error())
```

Comment:
format the error before returning the error to the user's `fmt.Errorf("error doing something: %v", err)`

Reply:
I import the func from other util package, is that ok?